source/admin_manual/logging.rst

```diff
@@ -6,15 +6,45 @@ Dovecot Logging
 
 **Dovecot always logs a detailed error message** if something goes wrong. If it doesn't, it's considered a bug and will be fixed. However, almost always the problem is that **you're looking at the wrong log file**; error messages may be logged to a different file than informational messages.
 
+By default Dovecot logs to syslog using mail facility. Dovecot can also be configured to log to files directly.
+
+Log File Location
+^^^^^^^^^^^^^^^^^
+
 You can find the log file locations by running:
 
 .. code-block:: none
 
    doveadm log find
 
-Dovecot log configuration is found in the ``conf.d/10-logging.conf`` file in the dovecot configuration folder (usually ``/etc/dovecot`` but may also be ``/usr/local/etc/dovecot``).
+Last Errors
+^^^^^^^^^^^
+
+You can easily print the last 1000 error messages of a running Dovecot:
+
+.. code-block:: none
+
+   doveadm log errors
+
+Changing Log File Paths
+^^^^^^^^^^^^^^^^^^^^^^^
+
+If you don't want to use syslog, you can make Dovecot log to files directly:
 
-By default Dovecot logs to syslog using mail facility. You can change the facility from :ref:`setting-syslog_facility` setting. The syslog configuration is often in ``/etc/syslog.conf`` or ``/etc/rsyslog*`` files. You can also configure Dovecot to write to log files directly, see below.
+.. code-block:: none
+
+   log_path = /var/log/dovecot.log
+   # If you want everything in one file, just don't specify info_log_path and debug_log_path
```

Comment:
This makes is sound like it may be a bad idea to use the same filename explicitly.  Maybe something like:

If info_log_path or debug_log_path is not set, the value from log_path is used.

Reply:
How about:

```
   # If not set, use the value from log_path
   info_log_path = /var/log/dovecot-info.log
   # If not set, use the value from info_log_path
   debug_log_path = /var/log/dovecot-debug.log
```

Reply:
sounds good